pkg/agent/controller/agent.go

```diff
@@ -26,8 +28,13 @@ import (
 const (
 	submarinerIpamGlobalIp = "submariner.io/globalIp"
 	serviceUnavailable     = "ServiceUnavailable"
+	invalidServiceType     = "UnsupportedServiceType"
+	clusterIP              = "cluster-ip"
 	originName             = "origin-name"
 	originNamespace        = "origin-namespace"
+	labelSourceName        = "lighthouse.submariner.io/sourceName"
+	labelSourceNamespace   = "lighthouse.submariner.io/sourceNamespace"
+	labelSourceCluster     = "lighthouse.submariner.io/sourceCluster"
```

Comment:
These are no longer needed.

Reply:
Plan is to replace the annotation with labels. Since annotation is used in many other places, this is to introduce labels without breaking annotation dependent code. EndpointSlice and Import aggregation will require labels.